spec/queries/keywords_query_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe(KeywordsQuery) do
+  describe '#call' do
+    context 'given NO keywords' do
+      it 'returns an empty list' do
+        keywords = described_class.new.call
+
+        expect(keywords).to(be_empty)
+      end
+    end
+
+    context 'given 5 keywords' do
+      it 'returns 5 keywords' do
+        Fabricate.times(5, :keyword)
```

Comment:
Always create the minimum of fixtures. A list starts at 2 so create only two keywords. More would be needed if we wanted to test the pagingation.

Reply:
Fixed in 4f4526bea3a181ad4a5dc98745b8aeb95387ab9f